client/js/analysis.js

```diff
@@ -10,10 +10,12 @@ window.onload = function() {
  */
 async function fetchResponse() {
   const param = document.getElementById('searchbar').value;
-  const response = await fetch(`/api/analysis?name=${param}`);
-  const value = await response.json();
-  console.log(value);
-  populationHandler(value);
+  try {
+    const response = await getData(`/api/analysis?name=${param}`);
+    populationHandler(response);
```

Comment:
nit: you shouldn't have this function in your try/catch. Try to make your try/catch as narrowly scoped as possible, otherwise this will just quietly swallow any errors from populationHandler. Better to do:
```
let response;
try {
  response = ...;
} catch (e) {
  ...
}
if (response) {
  populationHandler(response);
}
```